tools/internal_ci/linux/grpc_build_artifacts.cfg

```diff
@@ -16,7 +16,7 @@
 
 # Location of the continuous shell script in repository.
 build_file: "grpc/tools/internal_ci/linux/grpc_build_artifacts.sh"
-timeout_mins: 180
+timeout_mins: 240
```

Comment:
let's revert this change.

Reply:
Done.

Reply:
Looks like this change is still there? Let's revert as it really should be unnecessary.